_posts/2019-05-01-terrence-lee.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: post
+title:  "Terence Lee"
```

Comment:
Please use a title like "Keynote announcement" or something similar.

Reply:
how about that?